qkrehdwns/sql/create.sql

```diff
@@ -0,0 +1,15 @@
+use movie;
+
+create table user(
```

Comment:
테이블명을 지을 때는 보통 복수형으로 작성하는 편이에요!
무엇보다 user는 mysql에서 예약어로 지정되어 있어서 다른 이름을 사용하는 것이 오류를 방지할 수 있어요😊

```suggestion
create table users(
```